### PR TITLE
KOGITO-1791: Saving new file should add automatically an extension

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Review by jomarko — APPROVED** [review]
Works fine

**Comment by paulovmr** [comment]
This PR has commits from my [other opened PR](https://github.com/kiegroup/kogito-tooling/pull/140), just to avoid a conflict when the other is merged. When reviewing this, please consider only the [latest commit](https://github.com/kiegroup/kogito-tooling/pull/141/commits/de44d887a5e306334090284c3cd0bfaedda81e64).

File: packages/desktop/src/__tests__/backend/FileOperations.test.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+});
+
+jest.mock("../../backend/Menu", () => {
+  return {
+    Menu: jest.fn().mockImplementation()
+  };
+});
+
+describe("saveFile ipc event", () => {
+  test("SAVE_AS", () => {
```
Comment (tiagobento) [inline packages/desktop/src/__tests__/backend/FileOperations.test.tsx]: Can you please make this a descriptive string? This is not an ID or something like that, it's just a text to describe the test case.

**Review by tiagobento — APPROVED** [review]
Minor change request inline. Thanks for the PR @paulovmr !

**Review by jomarko — APPROVED** [review]
Thank you @paulovmr

**Review by ederign — APPROVED** [review]

**Pull request merged**